### PR TITLE
fix: getlines method runs wrong after compiled

### LINKED ISSUE #46: getLines method may run wrong after compiled
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.66 Safari/537.36

#### Steps to reproduce:

Use vue-clamp with content which makes `this.$refs.content.getClientRects()` return a list of length more than 1 
#### Actual results:

The `getLines` method in source code returns correct result. __But__ it returns wrong result after babel compiled. 

#### Expected results:

Get correct number of lines so that to be able to calculate correctly

#### Suggestion:

Babel compiles the `[...array]` into `[].concat(array)`.

> The concat method creates a new array consisting of the elements in the object on which it is called, followed in order by, for each argument, the elements of that argument (if the argument is an array) or the argument itself (if the argument is not an array). It does not recurse into nested array arguments. [more description](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat)

However `this.$refs.content.getClientRects()` returns a DomRectList(on Chrome) rather than a pure array. Then `concat` method views it as a non-array argument and  `[].concat(DomRectList)` is going to be return `[DomRectList]` and its length is going to be 1 forever which is not our expected result. 

Therefore, I suggest replacing`spread syntax` with `simple for-loop` in `getLines` method to make sure that the iterable object converts into array correctly.

### PR DESCRIPTION
**Pull request by mosuzi**
This will fix #46 @Justineo

### CODE REVIEW

File: src/components/Clamp.js [hunk 1]
```diff
@@ -125,8 +125,13 @@ export default {
...
     },
     getLines () {
+      const pureArray = []
+      const clientRects = this.$refs.content.getClientRects()
+      for (let i = 0; i < clientRects.length; i++) {
+        pureArray.push(clientRects[i])
+      }
       return Object.keys(
-        [...this.$refs.content.getClientRects()].reduce(
+        pureArray.reduce(
```
Comment (Justineo) [inline src/components/Clamp.js:134]: We can just use `Array.prototype.slice.call(clientRects)` here without the for loop.

Comment (mosuzi) [reply src/components/Clamp.js:134]: Right!

**Comment by mosuzi** [comment]
I repull request with for-loop replaced by slice.call @Justineo

**Review by Justineo — APPROVED** [review]

**Comment by hzmming** [comment]
请问下，既然这问题解决了，为什么npm包里的代码还是旧的？
版本0.3.1，是还没发布么？@Justineo

**Comment by mosuzi** [comment]
yes, the author hasn`t released another version

**Comment by Justineo** [comment]
Oh, missed this PR. I'll publish a new version soon.

**Comment by Justineo** [comment]
See `vue-clamp@0.3.2`.

**Pull request merged**